### PR TITLE
Content update to /deductions/medical/amount

### PR DESCRIPTION
**Pull request by nmakuch**
### Updating the content on the `/medical/deductions/amount` page
- Updated page content based on [this github issue](https://github.com/cds-snc/cra-claim-tax-benefits/issues/161)
- Updated translations
- Deleted unused translation keys

| Before | After |
|--------|-------|
|   Screen Shot 2019-10-07 at 16 56 18   |   Screen Shot 2019-10-07 at 16 57 24   |

### CODE REVIEW

**Comment by nmakuch** [comment]
> Please delete the sentence in bold "Enter the total cost of medical expenses"
> 
> Thanks!

We need to have a label over the input, I'd suggest that we maybe reword the chunk of text above to not be a call to action and to not use the word "Enter". 

I see what you were trying to do though :D

**Comment by YedidaZalik** [comment]
Please change page heading to "Medical costs"
Please change "Enter total medical costs" to "You may include medical costs for:"
Please change the heading above the box to "Enter total medical costs"

Thanks!

**Comment by nmakuch** [comment]
Screen Shot 2019-10-10 at 11 52 58

pertaining to Yedida's comment above, the heroku deployment was down
